Chore():database Setup

oliviertech212 opened the pull request:
#### what does this PR do:
Configure app to use Postgres DB and Sequelize ORM
#### Description:
this PR includes the following changes to the project
- installed database and Sequelize dependencies 
- configure Postgres DB, Sequelize  and connected database to the project
- created a user model and migrations
- updated README.md file with required setup instructions 
#### to test this please follow README.md file setup instructions

Commit:
Chore():database Setup
-install  all database & sequelize   dependencies
-create databases
-configure postgresdb and sequelize
-connect database to the project
-create user model and migrations
-create required setup in the Readme.md file
[#Finishes database Setup]

patsicko reviewed (changes requested):
In config.js file, the first line:  import dotenv from "dotenv";  is giving an error "  Cannot use import declarations in modules that export using CommonJS" 
you can use   const dotenv=require("dotenv"); either 
Otherwise you did great

wakaflorien reviewed (changes requested):
- Add migrate and seed scripts in package.json

Commit:
Fix():bugs
-change config.js to use require
-update user model to use camelcase
-create migrations and seeds
-update package.json to use migrate and seed scripts
-update README.md file
[#Finishes Fix]

mukunzidd commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Configure the app to use Postgres database](https://trello.com/c/tGbpQeAM/9-configure-the-app-to-use-postgres-database)
